vnext/Shared/HermesSamplingProfiler.cpp

```diff
@@ -4,25 +4,42 @@
 #include "pch.h"
 
 #include <chrono>
+#include <future>
 
 #include <hermes/hermes.h>
-#include <winrt/Windows.Storage.h>
 #include "Unicode.h"
 
+#ifdef WINRT
+#include <winrt/Windows.Storage.h>
+#else
+#include <ShlObj.h>
+#include <Shlwapi.h>
+#endif
+
 #include "HermesSamplingProfiler.h"
 
 namespace Microsoft::ReactNative {
 
 namespace {
 std::future<std::string> getTraceFilePath() noexcept {
+  std::ostringstream os;
+#ifdef WINRT
   auto hermesFolder = (co_await winrt::Windows::Storage::ApplicationData::Current().LocalFolder().CreateFolderAsync(
                            L"Hermes", winrt::Windows::Storage::CreationCollisionOption::OpenIfExists))
                           .Path();
+  os << Microsoft::Common::Unicode::Utf16ToUtf8(hermesFolder.c_str(), hermesFolder.size());
```

Comment:
nit - i think you can use winrt::to_string here. we should try to get rid of our own hand-rolled helper over time